include/printf.h

```diff
@@ -28,11 +28,18 @@
 #    define PRINTF_ENABLED
 #endif
 
+#if defined(X86_64_EFI_DEBUG) && !defined(PRINTF_ENABLED)
+#    define PRINTF_ENABLED
+#endif
+
 #ifdef PRINTF_ENABLED
 #   include <stdio.h>
 #   if defined(DEBUG_ZYNQ) && !defined(USE_QNX)
 #       include "xil_printf.h"
 #       define wolfBoot_printf(_f_, ...) xil_printf(_f_, ##__VA_ARGS__)
+#   elif defined(X86_64_EFI_DEBUG)
+/* XXX: %s argument will not work as EFI use widechar string */
```

Comment:
Please indent. What is the XXX? Can you change that?

Reply:
I use the XXX: to highlight the comment. I agree with you, a more descriptive tag is better. I changed it to NOTE: